lib/help.js

```diff
@@ -45,12 +45,19 @@ exports.customHelp = function(generator, usageFilePath, cmd) {
   }
 
   // Append USAGE file if any
-  var exists = g.f(usageFilePath);
-  if (exists) {
-    helpStr.push(exists);
+  var usageText = g.f(usageFilePath);
+  if (usageText !== usageFilePath) {
+    helpStr.push(usageText);
   }
 
   helpStr = helpStr.join('\n');
+
+  if (command === 'loopback-cli') {
+    return helpStr
+      .replace(/ (yo|slc) loopback:/g, ' lb ')
+      .replace(/ (yo|slc) loopback/g, ' lb');
```

Comment:
Before

```
Lorem ipsum yo loopback:model [options]
Lorem ipsum yo loopback [options]
```

```
Lorem ipsum lb model [options]
Lorem ipsum lb [options]
```

This second line (L58) handles the case of the default generator with no `:`.